source/code/plugins/out_oms_blob.rb

```diff
@@ -218,10 +220,17 @@ def handle_records(tag, records)
           # extra tags for CustomLog:
           # tags[4]: custom data type
           custom_data_type = tags[4]
-          suffix = Time.now.utc.strftime("d=%Y%m%d/h=%H/#{SecureRandom.uuid}")
+
+          # tags[5..-1]: monitoring file name
+          # concat all the rest parts with /
+          filePath = ''
```

Comment:
Use the [array join method.](http://ruby-doc.org/core-2.2.0/Array.html#method-i-join)
`filepath = tags[5..-1].join('/')`
Then on line 230, there is no need to remove the trailing slash.